admin-dev/themes/new-theme/js/pages/product/components/categories/index.js

```diff
@@ -36,30 +36,34 @@ const {$} = window;
 
 const ProductCategoryMap = ProductMap.categories;
 
+//@todo: filename is index.js but its a class CategoriesManager. Shouldn't we keep the filename categories-manager.js?
```

Comment:
It's a convention that when you use index.js it is the default file used when importing
So it allowed to do
`import CategoriesManager from '@pages/product/components/categories'`
instead of
`import CategoriesManager from '@pages/product/components/categories/categories-manager'`

That said I really don't care about it, if you feel like renaming it, be my guest ^^

Reply:
`categories` is not verbose enough to me, if it returns a CategoryManager class